clients/admin-ui/src/features/taxonomy/hooks.tsx

```diff
@@ -293,6 +316,28 @@ export const useDataUse = (): TaxonomyHookData<DataUse> => {
     };
   };
 
+  const handleToggleEnabled = async (
```

Comment:
I think you might be able to refactor this a bit such that there's less duplicate code. it looks like the only difference between these is that they each need a different `update...MutationTrigger`, so perhaps the update function can be passed in as an argument to a more general function, something like `handleEdit`. 

or maybe TS will be cranky about this sort of thing 🙃 worth trying though to cut down on duplicate code!

Reply:
I tried messing with it a bit but I don't think I can get it to work without giving the method an `any` type (and even then not easily), because its actual type is something insane.  If you'd like we could take a look at it together sometime tomorrow?  This was also templated after the existing create, edit, and delete functions which are all duplicated as well, so if there is a possible refactor we could do it for those too.

Reply:
ah got it, I vaguely recalled this being a tricky thing so I probably backed out and just copy/pasta'd 😄 don't worry about it then! I might tinker with it at some point but it's not a big deal 👍